akka-sample-mesos-scala/.gitignore

```diff
@@ -0,0 +1,17 @@
+*#
+*.iml
+*.ipr
+*.iws
+*.pyc
```

Comment:
Hmm I'm not sure we expect `*.pyc` files here :) - perhaps you might want to add those to your [global gitignore](https://davidwalsh.name/global-gitignore)? OK by me thoughl